src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -623,7 +623,7 @@ void AirspeedModule::select_airspeed_and_publish()
 			 * @description Previously selected sensor index: {1}, current sensor index: {2}.
```

Comment:
@MaEtUgR the description contains the details

Reply:
Aaaah, so that description also gets printed as "subtitle" and there the values are already magically inserted 💡 
Then sorry for any churn.